### ISSUE TITLE
Tournament endpoint: datetime format: consistency

### ISSUE THREAD

**Issue by smlbiobot** [cr-api/cr-api#146]
For consistency, it would be great if we can use UTC timestamp for all the date time fields.

```json
{
    "endedTime": "20171229T005055.000Z",
    "startTime": 1514505055,
    "createTime": 1514504131,
}
```

**Comment by Gr8z** [comment]
+1

**Comment by selfish** [comment]
Missed that, will fix.

**Comment by selfish** [comment]
Fixed in next deployment: `v7.7.2`
